matrix.yaml

```
@@ -3,7 +3,7 @@ CUDA_VER:
   - "11.8.0"
   - "12.0.1"
   - "12.2.2"
-  - "12.8.0"
+  - "12.9.0"
```

We need to produce both 12.8 and 12.9 images until we finish the migration.
```suggestion
  - "12.8.0"
  - "12.9.0"
```

---

Thanks Bradley! 🙏

Added this back and updated the PR OP to clarify this